cmake/projects/MySQL-client/hunter.cmake

```diff
@@ -19,9 +19,17 @@ hunter_add_version(
     SHA1
     2444586365c2c58e7ca2397d4617e5fe19f9f246
 )
-# CONFIGURATION_TYPES is set to Release, because Debug is bugged at the moment
-# see https://github.com/ruslo/hunter/issues/303
-hunter_configuration_types(MySQL-client CONFIGURATION_TYPES Release)
+
+hunter_add_version(
+    PACKAGE_NAME
+    MySQL-client
+    VERSION
+    "6.1.9"
+    URL
+    "https://github.com/hunter-packages/mysql-client/archive/v1.6.9.tar.gz"
+    SHA1
+    c1ba60a09238a2bc29fe79bc13609e0cfb6f489a
+)
 
 # https://github.com/ruslo/hunter/issues/705
 # hunter_cacheable(MySQL-client)
```

Comment:
It should be noted that the code using this is to setup merging the libraries into one. It might not affect the cmake config and therefore might be cache-able. I don't have a project fully setup to use this all just yet. I plan to get around to it in a few days and can test then.